README.md

```diff
@@ -1,2 +1,19 @@
-# imageAnalysisSystem
+# Low-Footprint Image Analysis System
 This system is a low-footprint versatile system involving hardware and software developments for recognition of objects and/or faces through machine learning algorithms.
+
+### Development environment
+To run the `dev` server:
+1. Navigate to the project directory in terminal
+2. Create a virtual environment using: `python3 -m venv env`
+3. Run `source env/bin/activate`. This will start the Python virtual environment.
+4. In the environment, run `python3 main.py` to run the server in the `main.py` file.
+5. Go to the port specified in the output
+6. To stop the environment, run `deactivate`
+
+### Team members
+| Name | Email  |
+|---|---|
+|  Rudy Patel | rutvik@ualberta.ca |
+|  Kaden Dreger | kaden@ualberta.ca |
+|  Jacob Paetsch | jpaetsch@ualberta.ca |
+|  Braden Stolte | bstolte@ualberta.ca |
```

Comment:
Just updated the README to be a bit cleaner. Also, I added the steps needed to start your own development environment!